### ISSUE TITLE
Add ability to update credentials

### ISSUE THREAD

**Issue by themccallister** [hashicorp/otto#118]
When using `otto infra` you are prompted to enter your AWS credentials. However, if you need to change those credentials, there does not appear to be a way to reset or override your current credentials.

**Comment by jmetzmeier** [comment]
Try entering nothing for the password. That should prompt for the credentials again.

**Comment by themccallister** [comment]
I assumed you were referent to 'password' as in the **Encrypted Credentials Password**.

Running `otto infra` prompts me for that Encrypted Credentials Password, entering a blank password prompts me to enter new credentials. However that seems counter intuitive, adding an incorrect password (blank or not) should invoke "invalid credentials" not a clean slate to overwrite the existing credentials.

**Comment by mitchellh** [comment]
Agreed. We added a `ValidateCreds` method that doesn't do much right now. We'll add this.
